src/NewToDo/NewToDo.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ClassNames from 'classnames';
+import './NewToDo.css';
+
+class NewToDo extends React.Component {
+  state = {
+    userId: 1,
+    id: this.props.toDoId + 1,
```

Comment:
generate an id for a new todo when you call addTodo method, there is no need to pass it through components back and forth

Reply:
not fixed again